### PR TITLE
Profile SignedIn testcases automation

### PR DESCRIPTION
**Pull request by Ayushee28**
Automated testcases from Profile SignedIn
Added ids for automation for some of the elements not having ids

@DaleMcGrew @fdiazq @mythrienjam @lavishaGit

### CODE REVIEW

File: tests/browserstack_automation/specs/SignedIn.js [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+      await driver.pause(waitTime);
+      await SignedIn.enterPhoneNumberInPhoneNumberInput();
+      await SignedIn.clickOnSendCodeOption();
+      await driver.pause(waitTime);
+      await SignedIn.enterVerificationCode();
+      await SignedIn.clickOnVerifyButtonForCodeVerification();
+      await driver.pause(waitTime);
+      await SignedIn.clickOnProfilePhotoPlaceholder();
+      await driver.pause(waitTime);
+      await expect(driver).toHaveUrl(expect.stringContaining('profile'));
```
Comment (lavishaGit) [inline tests/browserstack_automation/specs/SignedIn.js:44]: @Ayushee28 Just a suggestion correct me if I am wrong but there is no need to use driver—all commands and functionality are attached to the browser object by default.So you can use :
await expect(browser).toHaveUrl(expect.stringContaining('profile'));

Comment (Ayushee28) [reply tests/browserstack_automation/specs/SignedIn.js:44]: @lavishaGit I am not sure about that. I had tried with browser and it gives error sometimes for me. Only driver works fine. Also I saw others have used driver as well in the other specs so added this way

Comment (fdiazq) [reply tests/browserstack_automation/specs/SignedIn.js:44]: @Ayushee28 The use of driver is fine

**Comment by fdiazq** [comment]
@Ayushee28 The test cases look good. I recommend adding the new SignedIn.js to the config file

**Comment by Ayushee28** [comment]
> @Ayushee28 The test cases look good. I recommend adding the new SignedIn.js to the config file

Will make sure to add it in next pull request

**Pull request merged**